senderscore.py

```diff
@@ -3,25 +3,47 @@
 
 
 class SenderScore(BotPlugin):
-    """Plugin to show the mail score of the given ip"""
+    """Plugin to show the mail score of the given adress"""
 
     @botcmd
     def senderscore(self, msg, args):
-        """retrieve mail reputation usage !senderscore [IP]"""
-        return self.get_senderscores(args)
+        """retrieve mail reputation usage !senderscore [IP/Hostname]"""
+        try:
```

Comment:
Are you using a try/except to check if it is an ip? Why not an if? And are you sure this works?